tensorboard/pip_package/build_pip_package.sh

```diff
@@ -12,9 +12,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-set -e
-if [ -z "${RUNFILES}" ]; then
-  RUNFILES="$(CDPATH= cd -- "$0.runfiles" && pwd)"
+
+set -eu
+
+usage() {
+  cat <<EOF
+usage: build_pip_package OUTPUT_DIR
+
+Build TensorBoard Pip packages and store the resulting wheel files
+into OUTPUT (a \`.tar.gz\` path or directory).
```

Comment:
nit: I'd say ```a directory or a \`.tar.gz\` path``` to make it clear that that there is no need for the directory name to end in `.tar.gz` since it's otherwise strictly speaking ambiguous

Reply:
Agreed, absolutely. (I usually try to do this, but missed this one.)